core/controllers/question_editor.py

```diff
@@ -61,10 +61,10 @@ def post(self):
                 (question_dict['id'] is not None) or
                 ('question_state_data' not in question_dict) or
                 ('language_code' not in question_dict) or
-                (question_dict['version'] != 1)):
+                (question_dict['version'] != 0)):
```

Comment:
Why are we still dependent on the version number provided from the user to API? I this handler creates a new question then it should initiate its own version number and maybe send the question_id and version as a response?

[Wanted to check the specific reason for API to depend on this value?]

Reply:
I had mentioned it in the PR description. Basically, since the entire question is created, we are using the QuestionObjectGactory to represent it before creation in the frontend and it has the version field as well.

Reply:
We are passing null for the id as it's not yet created so why can't we do the same for the version i.e, have it 0 or null initially and pass the version number and id through the post response?

Reply:
Yeah, that's why I am now sending 0, since we can define the version during creation, but question ID can only be created in the backend.